recipes/lapack/build.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+mkdir build
+cd build
+
+# CMAKE_INSTALL_LIBDIR="lib" suppresses CentOS default of lib64 (conda expects lib)
+
+cmake \
+  -DCMAKE_INSTALL_PREFIX=${PREFIX} \
+  -DCMAKE_INSTALL_LIBDIR="lib" \
+  -DBUILD_TESTING=OFF \
+  -DBUILD_SHARED_LIBS=ON \
+  -DLAPACKE=ON \
+  ..
+
+make -j2
```

Comment:
Sure.  You're welcome to propose changes to conda build.  The code you might want to start with is at https://github.com/conda/conda-build/blob/69cba12cfcfb585ff1167c95a8b58307acd0ca18/conda_build/environ.py#L258

The tricky part will be where the limit gets defined, when the limit is other than the system's reported number of processors.